toolkit/tools/internal/sliceutils/sliceutils.go

```diff
@@ -59,11 +59,12 @@ func PackageVerMatch(expected, given interface{}) bool {
 	return reflect.DeepEqual(expected.(*pkgjson.PackageVer), given.(*pkgjson.PackageVer))
 }
 
-// SetToSlice converts a map[T]bool to a slice containing the map's keys.
-func SetToSlice[T comparable](inputSet map[T]bool) []T {
+// SetToSlice converts a map[K]bool to a slice containing the map's keys, iff the key's value is true.
+func SetToSlice[K comparable](inputSet map[K]bool) []K {
 	index := 0
-	outputSlice := make([]T, len(inputSet))
+	outputSlice := make([]K, len(inputSet))
 	for element, elementInSet := range inputSet {
+		// Add key to slice if value is true
```

Comment:
Seems unnecessary - the variable's name is already explicit enough. Too many comments may muddy the code instead of adding clarity. I'd strongly suggest saving them only for cases where the logic is too complex to be explained through clear code or well-named variables.
```suggestion
```